tests/data/scad/experimental/object/object-tests.scad

```diff
@@ -134,7 +134,17 @@ testEq( object( $fs = 42 ).$fs, 42, "using config names as field is ok");
 testEq( object( this = 42 ).this, 42, "also this");
 testEq( object( this = 42, f=function(this) this.this ).f(), 42, "but functions see the this object");
 
-
+{
```

Comment:
This line doesn't create a new scope ([see parser.y](https://github.com/openscad/openscad/blob/ce6fcc5480eebe5a421c67aee03f39dc352d3327/src/core/parser.y#L187)) so I wouldn't use it to remain consistent with the remainder of the tests in this file and not be misleading (I have to remind myself it doesn't create new scope every time I see it).

Reply:
I suggest after this is in the code base you make a PR with these suggestions.